iml-services/iml-device/src/lib.rs

```diff
@@ -457,6 +458,30 @@ pub fn find_targets<'a>(
             uuid: fs_uuid.into(),
             mount_path: Some(mntpnt.0.to_string_lossy().to_string()),
         })
+        .collect();
```

Comment:
Is the collect necessary? Couldn't it go straight into a fold?

Reply:
It could, but up to this point there seemed to be one concept per iterator so I decided to keep it that way.